[2020-09-28 10:11:55,865] [view.py:57] [view:get_predict_flow] [INFO]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:请求成功，任务开始启动
[2020-09-28 10:12:19,022] [view.py:57] [view:get_predict_flow] [INFO]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:请求成功，任务开始启动
[2020-09-28 10:12:36,443] [view.py:57] [view:get_predict_flow] [INFO]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:请求成功，任务开始启动
[2020-09-28 10:13:01,941] [view.py:57] [view:get_predict_flow] [INFO]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:请求成功，任务开始启动
[2020-09-28 10:13:38,708] [view.py:57] [view:get_predict_flow] [INFO]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:请求成功，任务开始启动
[2020-09-28 10:15:27,427] [view.py:55] [view:get_predict_flow] [INFO]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:请求成功，任务开始启动
[2020-09-28 10:33:01,883] [view.py:55] [view:get_predict_flow] [INFO]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:请求成功，任务开始启动
[2020-09-28 10:34:24,069] [view.py:80] [view:get_predict_flow] [ERROR]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:'None of [0] are in the columns'
[2020-09-28 10:34:27,751] [view.py:80] [view:get_predict_flow] [ERROR]- [d50aa359-bd23-4037-958b-d0c55ed408aa]:'None of [0] are in the columns'
[2020-09-28 10:51:52,518] [view.py:55] [view:get_predict_flow] [INFO]- [a042a999-0df2-4dc0-a0bd-3c8a94216f14]:请求成功，任务开始启动
[2020-09-28 10:56:11,181] [view.py:55] [view:get_predict_flow] [INFO]- [a7aa8426-8baa-4b4f-aeef-bac68f409c09]:请求成功，任务开始启动
[2020-09-28 11:01:09,182] [view.py:55] [view:get_predict_flow] [INFO]- [a7aa8426-8baa-4b4f-aeef-bac68f409c09]:请求成功，任务开始启动
[2020-09-28 11:01:29,439] [view.py:55] [view:get_predict_flow] [INFO]- [a7aa8426-8baa-4b4f-aeef-bac68f409c09]:请求成功，任务开始启动
[2020-09-28 11:02:29,381] [view.py:55] [view:get_predict_flow] [INFO]- [9c2c9178-0ac9-4a22-bba9-8ebe4e92c2bb]:请求成功，任务开始启动
[2020-09-28 11:05:20,805] [view.py:55] [view:get_predict_flow] [INFO]- [9c2c9178-0ac9-4a22-bba9-8ebe4e92c2bb]:请求成功，任务开始启动
[2020-09-28 11:05:42,895] [view.py:55] [view:get_predict_flow] [INFO]- [9c2c9178-0ac9-4a22-bba9-8ebe4e92c2bb]:请求成功，任务开始启动
[2020-09-28 11:18:17,884] [view.py:55] [view:get_predict_flow] [INFO]- [9c2c9178-0ac9-4a22-bba9-8ebe4e92c2bb]:请求成功，任务开始启动
[2020-09-28 11:18:38,157] [view.py:55] [view:get_predict_flow] [INFO]- [9c2c9178-0ac9-4a22-bba9-8ebe4e92c2bb]:请求成功，任务开始启动
[2020-09-28 11:19:39,499] [view.py:55] [view:get_predict_flow] [INFO]- [a7aa8426-8baa-4b4f-aeef-bac68f409c09]:请求成功，任务开始启动
[2020-09-28 11:19:51,925] [view.py:55] [view:get_predict_flow] [INFO]- [a7aa8426-8baa-4b4f-aeef-bac68f409c09]:请求成功，任务开始启动
[2020-09-28 11:21:29,592] [view.py:55] [view:get_predict_flow] [INFO]- [a042a999-0df2-4dc0-a0bd-3c8a94216f14]:请求成功，任务开始启动
[2020-09-28 11:21:39,389] [view.py:55] [view:get_predict_flow] [INFO]- [a042a999-0df2-4dc0-a0bd-3c8a94216f14]:请求成功，任务开始启动
[2020-09-28 16:47:19,552] [view.py:54] [view:get_predict_flow] [INFO]- [9a4dc045-f58f-4c02-95b1-e63c7e376d93]:请求成功，任务开始启动
[2020-09-28 16:48:08,263] [view.py:54] [view:get_predict_flow] [INFO]- [9a4dc045-f58f-4c02-95b1-e63c7e376d93]:请求成功，任务开始启动
[2020-09-28 16:49:20,442] [view.py:54] [view:get_predict_flow] [INFO]- [81b03e93-67e8-49cf-9739-a81a02c84a17]:请求成功，任务开始启动
